### ISSUE TITLE
Current Reality Capture Nerfstudio Import process producing very bad quality for Splatfacto.

### ISSUE THREAD

**Issue by Maxiviper117** [nerfstudio-project/nerfstudio#3118]
**Describe the bug**
The current nerfstudio instructions on importing from Reality Capture at: https://docs.nerf.studio/quickstart/custom_dataset.html#realitycapture don't produce expected results.

I have a suspicion that the issue may be in using the exported CSV from `Internal/External camera parameters` export option.

~Does the CSV export take into account the **distortion** the images undergo as Reality Capture optimizes the alignment?
Or is that do in Nerfstudio?~

This problem does not happen if using the colmap `.txt` and `colmap.db` format generated from the bundler `.out`  and imagelist `.lst` exports with undistorted images, using the process shown here https://gist.github.com/jo-chemla/258e6e40d3d6c2220b29518ff3c17c40

Then taking that colmap format and using the [instant-ngp](https://github.com/NVlabs/instant-ngp/blob/de507662d4b3398163e426fd426d48ff8f2895f6/scripts/colmap2nerf.py) `colmap2nerf.py` script to convert to a `transforms.json` file with this section of that script disabled in the `colmap2nerf.py` script:

```python
if __name__ == "__main__":
	args = parse_args()
	#if args.video_in != "":  <---
	#	run_ffmpeg(args)  <---
	# if args.run_colmap:    <---
	#	run_colmap(args)  <---
	AABB_SCALE=int(args.aabb_scale)
	SKIP_EARLY=int(args.skip_early)
	IMAGE_FOLDER=args.images
```

Using this newely generate transforms.json file with the undistorted images, results is far greater quality (as shown in the screenshot) below.

**To Reproduce**

**Expected behavior**
Expecting quality to not be this bad.

**Screenshots**
![image](https://github.com/nerfstudio-project/nerfstudio/assets/11472492/39c4bc5d-335b-4c12-8cb7-2d975ca6095d)

**Additional context**

**Comment by jb-ye** [comment]
The converter code was created before splatfacto. I believe a possible issue could be related to porting point cloud from reality capture to ply data with proper coordinate transform. 

Just want to confirm if nerfacto works for your reality capture data? If the quality is also bad, then maybe something has been broken.

**Comment by Maxiviper117** [comment]
I havent tried with Nerf but my only interest is with Gaussian, I doubt the Nerf would fare much better if the conversion is incorrect to begin with. After looking at the function that handles the conversion, I can see it's definitely overlooking a few things.

![image](https://github.com/nerfstudio-project/nerfstudio/assets/11472492/9eca02a0-aa86-4610-85b2-d894f5cc3a00)


Reality Capture has documentation regarding the [Principal Point](https://rchelp.capturingreality.com/en-US/tools/undistort.htm#:~:text=Alternatively%2C%20you%20can%20define%20the,See%20also:) and [Distortion Models.](https://rchelp.capturingreality.com/en-US/appbasics/settings_distortion_models.htm)

**Comment by jb-ye** [comment]
@Maxiviper117 I am not a user of reality capture, if you know how to make it work with splatfacto, please send a PR fixing that.

**Comment by Maxiviper117** [comment]
I'm not qualified to do that, closing issue hopefully someone else can help in future :(

**Comment by Maxiviper117** [comment]
Re-opening to leave it another good Samaritan.

**Comment by jb-ye** [comment]
<img width="664" alt="Screenshot 2024-05-08 at 12 47 11 PM" src="https://github.com/nerfstudio-project/nerfstudio/assets/132313008/2f3463dc-f58f-4e61-a2c4-9c8d24a7f991">
I was able to fix this issue using #3131
